git-hooks/pre-commit

```diff
@@ -74,8 +74,26 @@ check_make_lint() {
     exit 1
 }
 
+#
+# Don't allow path.Join()
+#
+check_path_Join()
+{
+    for file in $(git diff --cached --name-only --diff-filter=ACM | grep '.go$' | grep -v go/vendor/ | grep -v vendor/)
+    do
+    pathjoins=$(grep -E ' path\.Join\(')
```

Comment:
you might not always have a space before it. Does `'\bpath\.Join\('` work instead?  `\b` means "word boundary"